manifest.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Tab Size on GitHub",
-	"version": "1.1.2",
+	"version": "1.1.3",
```

Comment:
These blocks of code are what this PR applies to.  They're new with the new review/single comment feature introduced by github.